regulations/static/regulations/js/source/views/main/main-view.js

```diff
@@ -32,74 +32,55 @@ var MainView = Backbone.View.extend({
-            childViewOptions.subContentType = appendixOrSupplement;
-        }
-
-        // find search query
-        if (this.contentType === 'search-results') {
-            childViewOptions.params = URI.parseQuery(window.location.search);
-            childViewOptions.query = childViewOptions.params.q;
-        }
-
-        if (this.contentType === 'landing-page') {
-            DrawerEvents.trigger('pane:change', 'table-of-contents');
-        }
-
-        // we don't want to ajax in data that the page loaded with
-        childViewOptions.render = false;
-
-        if (this.sectionId && this.modelmap[this.contentType]) {
-            // store the contents of our $el in the model so that we
-            // can re-render it later
-            this.modelmap[this.contentType].set(this.sectionId, this.$el.html());
-            childViewOptions.model = this.modelmap[this.contentType];
-        }
-
-        if (this.contentType && typeof this.viewmap[this.contentType] !== 'undefined') {
-            // create new child view
-            this.childView = new this.viewmap[this.contentType](childViewOptions);
-        }
-
-        this.sectionFooter = new SectionFooter({el: this.$el.find('.section-nav')});
+      this.dataTables = null;
+
+      if (Router.hasPushState) {
+        this.listenTo(MainEvents, 'search-results:open', this.createView);
+        this.listenTo(MainEvents, 'section:open', this.createView);
+        this.listenTo(MainEvents, 'diff:open', this.createView);
+        this.listenTo(MainEvents, 'breakaway:open', this.breakawayOpen);
+        this.listenTo(MainEvents, 'section:error', this.displayError);
+      }
+      this.listenTo(MainEvents, 'section:resize', this.applyTablePlugin);
+      this.listenTo(MainEvents, 'section:setHandlers', this.setHandlers);
+
+      this.$topSection = this.$el.find('section[data-page-type]');
+
+      // which page are we starting on?
+      this.contentType = this.$topSection.data('page-type');
+      // what version of the reg?
+      this.regVersion = Helpers.findVersion(Resources.versionElements);
+      // what section do we have open?
+      this.sectionId = this.$topSection.attr('id') ||
+        this.$topSection.find('section[id]').attr('id');
+      this.docId = $('#menu').data('doc-id');
+      this.cfrTitle = $('#menu').data('cfr-title-number');
+
+      if (this.sectionId && this.modelmap[this.contentType]) {
+        // store the contents of our $el in the model so that we
+        // can re-render it later
+        this.modelmap[this.contentType].set(this.sectionId, this.$el.html());
+        this.childModel = this.modelmap[this.contentType];
+      }
+
+      var options = {
+        el: '#' + this.sectionId,
+        subContentType: this.isAppendixOrSupplement(),
+        render: false
+      };
+
+      if (this.contentType === 'search-results') {
+        options.params = URI.parseQuery(window.location.search);
+        options.query = options.params.q;
+      }
```

Comment:
I don't think so--we call `setChildOptions` before instantiating a new child view, meaning we haven't pushed the new query parameters to the window history yet.